spyder/config/main.py

```diff
@@ -170,7 +170,8 @@
               'in_prompt': '',
               'out_prompt': '',
               'show_elapsed_time': False,
-              'ask_before_restart': True
+              'ask_before_restart': True,
+              'subprocess_windows': False

```

Comment:
Should this be `True` by default? Users using `shell=True` expect this, no @ccordoba12 ?

Reply:
This is False because there are libraries like Pyomo that generate a lot of Command Prompts while running, and that's extremely annoying for Windows users.

Reply:
That could be a great comment :-)

``` 
#This is False because there are libraries like Pyomo that generate a lot of Command Prompts while running, and that's extremely annoying for Windows users.
```

Reply:
Ok, that's true :+1:. @dalthviz, please add my comment above that option, and also change its name to `show_cmd_windows`.

Reply:
So @dalthviz, please change this option to `'hide_cmd_windows': True` instead.